src/DocRaptor.php

```diff
@@ -71,10 +67,19 @@ public function setDocumentUrl($document_url)
     /**
      * @param string $document_type
      * @return $this
+     * @throws Exception
      */
     public function setDocumentType($document_type)
     {
+        $allowedValues = array('xls', 'xlsx', 'pdf');
```

Comment:
I don't have any idea what adoption is for versions. This is fine as-is if 5.6 is a non-starter for a lot of people.